### PR TITLE
Added logging of export time in milliseconds

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#445

Logs screenshot:
Screenshot 2021-10-01 at 13 04 08

### CODE REVIEW

File: exporter/terra/terra_listener.py [hunk 1]
```diff
@@ -73,8 +72,12 @@ def _experiment_message_handler(self, body: str, msg: Message):
         try:
             exp = ExperimentMessage.from_dict(json.loads(body))
             self.logger.info(f'Received experiment message for process {exp.process_uuid} (index {exp.experiment_index} for submission {exp.submission_uuid})')
+            start_time = datetime.datetime.now()
```
Comment (amnonkhen) [inline exporter/terra/terra_listener.py]: A much cleaner and reusable approach would be to add a timing decorator to `terra_exporter.export()`

**Review by amnonkhen — CHANGES_REQUESTED** [review]

**Review by jacobwindsor — COMMENTED** [review]
Does this log the UUID with the export time? No right? I think it would be useful to have project/submission on all log messages from the exporter

**Comment by aaclan-ebi** [comment]
@jacobwindsor yes, it logs the function parameters and  submission UUID and project UUID is there. I've put the screenshot of logs above

> it would be useful to have project/submission on all log messages from the exporter

could be in a separate ticket/PR

File: exporter/terra/terra_listener.py [hunk 2]
```diff
@@ -1,18 +1,17 @@
+import datetime
```
Comment (ke4) [inline exporter/terra/terra_listener.py:1]: I believe this import is not used

**Review by ke4 — APPROVED** [review]
👍

**Review by jacobwindsor — APPROVED** [review]

**Comment by aaclan-ebi** [comment]
I forgot to set the target branch to dev so I've merged the branch separately thru the command line.

**Pull request merged**